assets/src/js/admin/onboarding-wizard/app/steps/location/index.js

```diff
@@ -18,7 +18,7 @@ const Location = () => {
 	return (
 		<div className="give-obw-location">
 			<h2>{ __( 'Where are you fundraising?', 'give' ) }</h2>
-			<Card>
+			<Card padding="20px 40px 60px 40px">
```

Comment:
Inline styles should probably be extracted into a stylesheet.

Reply:
Because cards don't use consistent padding in the Figma designs, I opted to use a `padding` prop that can be passed to set specific padding on a card by card basis (to avoid adding style rules for cards to components that use them and want to override card padding). Open to suggestions on this approach!

Reply:
Oh, ok. So uniformity is enforced through padding. We may want to consider setting a fixed (or relative) size for the card component and then align the contents with flexbox.

Reply:
Deferred to #4955